src/pages/options/options.js

```diff
@@ -183,10 +183,17 @@ const optionsFooter = css`
 `
 
 const OptionsApp = () => {
-  const [sitesTwitter, setSitesTwitter] = useState(getBool(getSetting('sites_twitter')))
-  const [darkMode] = useState(getBool(getSetting('darkMode')))
-  // const [accessToken] = useState(getSetting('access_token'))
-  // const [userName] = useState(getSetting('username'))
+  const [sitesTwitter, setSitesTwitter] = useState()
+  const [darkMode, setDarkMode] = useState()
+  const [accessToken, setAccessToken] = useState()
+  const [userName, setUserName] = useState()
+
+  useEffect( async ()=>{
+    setSitesTwitter(getBool(await getSetting('sites_twitter')))
+    setDarkMode(getBool(await getSetting('dark_mode')))
+    setAccessToken(await getSetting('access_token'))
+    setUserName(await getSetting('username'))
+  })
```

Comment:
Need to add in an empty array to make sure this effect only runs when the component is loaded right?

`}, [])`

Reply:
I second this. The array makes it behave like `componentDidMount` while omitting the array makes it behave like `componentDidUpdate`

Reply:
Ack! Yes! If I had a dollar...